go.mod

```diff
@@ -5,15 +5,15 @@ go 1.14
 require (
 	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
 	github.com/golang/protobuf v1.4.2
-	github.com/google/go-cmp v0.4.1
+	github.com/google/go-cmp v0.5.0
 	github.com/kylelemons/godebug v1.1.0
 	github.com/openconfig/gnmi v0.0.0-20200508230933-d19cebf5e7be
 	github.com/openconfig/goyang v0.2.2
```

Comment:
Since you're already upgrading some dependencies, could you run `go get -u ./...` in ygot to update dependencies such as goyang as well?